### PR TITLE
Analyzer: Fix bug in python3.8 due to pandas api change

### PR DESCRIPTION
**Pull request by yukirora**
**Description**
Analyzer: Fix bug in python3.8 due to pandas api change.

**Major Revision**
- force check numeric only in dataframe for analysis
- dataframe.append -> pd.concat
- pd.ExcelWriter.save() -> pd.ExcelWriter.close()

### CODE REVIEW

**Review by abuccts — COMMENTED** [review]
maybe fix the pandas to specific version? they're breaking changes starting from [2.0.0](https://pandas.pydata.org/docs/whatsnew/v2.0.0.html#backwards-incompatible-api-changes)

**Comment by yukirora** [comment]
> maybe fix the pandas to specific version? they're breaking changes starting from [2.0.0](https://pandas.pydata.org/docs/whatsnew/v2.0.0.html#backwards-incompatible-api-changes)

sure, updated to fix version to <2.0.0

File: superbench/analyzer/result_summary.py [hunk 1]
```diff
@@ -217,7 +217,7 @@ def output_summary_in_excel(self, raw_data_df, summary, output_path):
                 file_handler.merge_column_in_excel(worksheet, row, 1)
             else:
                 logger.error('ResultSummary: excel_data_output - summary is empty.')
-            writer.save()
+            writer.close()
```
Comment (cp5555) [inline superbench/analyzer/result_summary.py:220]: So, it will save the excel when closing?

File: setup.py [hunk 2]
```diff
@@ -163,7 +163,7 @@ def run(self):
         'numpy>=1.19.2',
         'omegaconf==2.0.6',
         'openpyxl>=3.0.7',
-        'pandas>=1.1.5',
+        'pandas>=1.1.5, <2.0.0',
```
Comment (cp5555) [inline setup.py:166]: Why not support >2.0.0 directly?

File: superbench/analyzer/result_summary.py (hunk 1 above)
Comment (yukirora) [reply superbench/analyzer/result_summary.py:220]: yes, it's synonym for save, they replace save to close in latest version to make it more file-like

File: setup.py (hunk 2 above)
Comment (yukirora) [reply setup.py]: updated

**Pull request merged**